lib/retirement_swap/swap_algorithm.rb

```diff
@@ -5,8 +5,7 @@ def initialize(storage, panoptes)
       @panoptes = panoptes
     end
 
-    def process(hash)
-      classification = Classification.new(hash)
+    def process(classification, agent, subject)
```

Comment:
Unused method argument - `subject`. If it's necessary, use `_` or `_subject` as an argument name to indicate that it won't be used.<br>Assignment Branch Condition size for process is too high. [26.27/15]